tests/preprocessing/test_util.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+
+import pytest
+
+from trackintel.preprocessing.util import calc_temp_overlap
+
+
+@pytest.fixture
+def time_1():
+    return datetime.datetime(year=1, month=1, day=1, hour=0, minute=0, second=0)
+
+
+@pytest.fixture
+def one_hour():
+    return datetime.timedelta(hours=1)
+
+
+class TestCalc_temp_overlap():
+    def test_same_interval(self, time_1, one_hour):
```

Comment:
The codes are pretty self-explanatory, still would be nice to have one-line docstring (for consistency).

Reply:
👍